src/qc_tool/file_handler.py

```diff
@@ -32,7 +34,14 @@ def __init__(
         self._file_button.on_click(self._load_file_callback)
 
         self._save_as_button = Button(label="Save as...")
-        self._save_as_button.on_click(self._save_file_as_callback)
+        self._save_as_button.on_click(
+            lambda: self._save_file_as_callback(self._external_save_file_callback)
```

Comment:
Det funkar absolut att göra såhär med lambda men en enklare variant skulle kunna vara att skicka in bara `self._save_file_as_callback`och `self._save_changes_as_callback` och om de har gemensamma saker de vill göra kan de båda kalla en funktion som gör det.

Reply:
Ja, här hade jag svårt att förstå hur det skulle fungera. self._save_file_as_callback och self._save_changes_as_callbac gör ju samma men med olika dataframes och namn på knapparna. Jag skulle vilja att det fanns en save_file_as_callback som både kallade på med olika input för vilken dataframe som ska sparas.